html/janus.js

```diff
@@ -1,3 +1,5 @@
+"use strict";
```

Comment:
I'm not aware of any modern browser / runtime not supporting strict mode.
What will be impacted is the way you'll write the code though ;-)

Reply:
'use strict' itself has been supported by all browsers since 2011/2012 (so even if supporting, say, IE10 were a goal it'd work); of course that doesn't necessarily mean it'll Just Work™ with the code as is. ^_^

Reply:
What I want to ensure is that demos won't start breaking because of this, as I've written them the same way at the time.

Reply:
I have been through all the demos and can confirm that they still (at least superficially) work in Firefox and do not log any exceptions. I have also added another build step to make npm produce a CommonJS file so as to confirm that works. Our application also continues to work.

I have not spotted the code doing anything forbidden by `use strict`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode